recipes/r-archr/meta.yaml

```diff
@@ -0,0 +1,124 @@
+{% set version = '1.0.2' %}
+{% set github = "https://github.com/GreenleafLab/ArchR" %}
+{% set commit = 'd9e741c980c7c64e5348c97a74d146cc95f8ba76' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-archr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: {{ github }}/archive/{{ commit }}.zip
+  sha256: aef7bd08d55494348cac8b9e0645a71a22a41dacbb7a7843cd939f1f31ee6d87
+
+build:
+  merge_build_host: True  # [win]
+  number: 8
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  run_exports:
+    - {{ pin_subpackage("r-archr", max_pin="x.x") }}
+
+requirements:
+  build:
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - {{ compiler('c') }}          # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
+    - {{ compiler('cxx') }}        # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
```

Comment:
- stdlib("c") ?

Reply:
I think that is still only implemented on Conda Forge. Gave me [a linting error here](https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=71416&view=logs&j=eb1ca712-fec5-5d93-deb0-40983fe61f2f&t=c9ebc308-e058-58af-a4f4-866fa7649dbb&l=39).